README.md

```diff
@@ -51,6 +51,12 @@ end
   !array.include?(element)
 ```
 
+- No spaces inside hash braces `{`, `}`
+
+```ruby
+  some_hash = {one: 1, two: 2, three: 3}
```

Comment:
The example could go between line 45 and 46, saving us a redundant section

Reply:
Moved that now